src/meshing/TecplotWriter.cc

```diff
@@ -499,8 +507,17 @@ bool WriteSingleDevice(const std::string &dname, std::ostream &myfile, const std
           ConstEdgeModelPtr emp = eit->second;
 
           //// Only save the fields that are "included"
-          if (!std::regex_match(name, std::regex(include))) {
-            continue;
+          bool is_included = false;
+          is_included = include.empty(); // Assume included if no patterns are specified
+          for (const std::string& pattern : include) {
```

Comment:
I haven't worked too much with C++ regex.  Is the style similar enough to Python that users will not have any problems?

Reply:
I think it is similar enough, or at least nothing a quick search cannot clarify. Because of the list approach, a user can also simply just list the names of all the fields individually if they wish.